model/host/db.go

```diff
@@ -469,7 +489,6 @@ func NeedsReprovisioningLocked(currentTime time.Time) bson.M {
 	cutoffTime := currentTime.Add(-MaxLCTInterval)
 	return bson.M{
 		StatusKey:               evergreen.HostProvisioning,
-		StartedByKey:            evergreen.User,
```

Comment:
Same question about why we are including spawn hosts.

Reply:
Same response. Spawn hosts aren't immune to this requirement.